packages/storybook/stories/Adapters/Adapters.stories.mdx

```diff
@@ -31,6 +32,8 @@ Teams Meeting information can be acquired through [Microsoft Graph](https://docs
 
 <Source code={CallAdapterExampleText} />
 
+<MessageBar>Note: The `displayName` in Call Composite can be a maximum of 256 characters.</MessageBar>
```

Comment:
Should we add this as a tsdoc comment in `AzureCommunicationCallAdapterArgs` and i stateful uses in storybook e.g.
OverviewStateful.stories.mdx:
![image](https://user-images.githubusercontent.com/2684369/144663202-bbff4912-14a4-464b-a937-4bf12fcccc1b.png)
(couple other places of example snippets)

Reply:
Good idea.